### DISCUSSION TITLE
I have studio data  (clean audio + text) per speakers, Which model is the best?

### DISCUSSION THREAD

**Discussion by cod3r0k** [idiap/coqui-ai-TTS#125] [Q&A] [answered]
I have studio data (clean audio + text) per speaker. Which model is the best?

**Comment by eginhard** [comment, marked as answer]
Coqui includes a large number of pretrained models that you can evaluate for your use case, e.g. compare the different English models trained on LJSpeech. Note that "best" can mean many different things (naturalness, intelligibility, speed, efficiency, ...)

**Reply by cod3r0k** [reply]
Dear @eginhard ,

Do we have any model that combines naturalness, speed, and efficiency? If yes, could you please recommend it? Additionally, do we have any model that supports emotion recognition? Is there an "all-in-one" model available as well?

Thank you.

**Reply by pivolan** [reply]
In my opinion and experiments, vits, and xtts_v2 is the best models. Xtts_v2 much more difficult to fine tune.
